packages/replay-next/components/console/LoggablesContext.tsx

```diff
@@ -126,8 +131,14 @@ export function LoggablesContextRoot({
 
   // We may suspend based on this value, so let's this value changes at sync priority,
   let exceptions: UncaughtException[] = EMPTY_ARRAY;
-  if (showExceptions) {
-    exceptions = getExceptionsSuspense(client, focusRange);
+  if (focusRange && showExceptions) {
+    try {
+      exceptions = getExceptionPointsSuspense(client, toPointRange(focusRange));
+    } catch (errorOrPromise) {
+      if (isThennable(errorOrPromise)) {
+        throw errorOrPromise;
+      }
+    }
```

Comment:
nit: Consider using `createInfallibleSuspenseCache` for this?

Outside of the component (module level)
```js
const infallibleExceptionPointsSuspense = createInfallibleSuspenseCache(getExceptionPointsSuspense);
```

Inside of the component
```ts
let exceptions: UncaughtException[] = EMPTY_ARRAY;
if (focusRange && showExceptions) {
  exceptions =
    infallibleExceptionPointsSuspense(client, toPointRange(focusRange)) ??
    EMPTY_ARRAY;
}
```